scripts/check/check_cpufreq.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import re
+import argparse
+
+sysfs_cpu_online = "/sys/devices/system/cpu/online"
+
+def get_file_content(path, default=None, strip=True):
+    data = default
+    if os.path.exists(path) and os.access(path, os.R_OK):
+        try:
+            try:
+                datafile = open(path)
+                data = datafile.read()
+                if strip:
+                    data = data.strip()
+                if len(data) == 0:
+                    data = default
+            finally:
+                datafile.close()
+        except Exception:
+            pass
+    return data
+
+
+def parse_opts():
+    parser = argparse.ArgumentParser(
+        description="Check Linux system CPUfreq governor.")
+    parser.add_argument("--available_governors", action="store_true", default=False,
+                        help="Show the CPUfreq governors available in the kernel.")
+    parser.add_argument("--current_governor", action="store_true", default=False,
```

Comment:
I think it's better to use '-' instead of '_' in command line arguments. (the variable name will still be foo_bar in argparse, while argument is `--foo-bar`)